pkg/sizing/algorithm.go

```diff
@@ -33,7 +34,7 @@ func calculateClusterSize(nt NodeType, bytesDayIngest float64, qperf QueryPerf)
 	replicasOnLastNode := math.Mod(numWriteReplicasNeeded, writeReplicasPerNode)
 
 	coresOnLastNode := 0.0
-	if replicasOnLastNode >= 0.0 {
+	if replicasOnLastNode > 0.0 {
```

Comment:
This was the bug in the algorithm. Glad the test caught it.

Reply:
nice!